src/dlite-entity.c

```diff
@@ -716,7 +716,13 @@ DLiteInstance *dlite_instance_get(const char *id)
 
     /* Set read-only as default mode (all drivers should support this) */
     if (!options) options = "mode=r";
-    if ((s = dlite_storage_open(driver, location, options))) {
+    ErrTry:
+      s = dlite_storage_open(driver, location, options);
+    ErrCatch(dliteStorageOpenError):  // suppressed error
```

Comment:
As discussed some way of tracking/logging these errors would be useful

Reply:
Added issue #328 to address your previous comment. I guess that will address this one as well.